specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -896,6 +899,9 @@ union AcsEmailDeliveryReportStatus {
 model AcsEmailDeliveryReportStatusDetails {
   /** Detailed status message */
   statusMessage?: string;
+  
+  /** Recipient mail server */
+  recipientMailServer?: string;
```

Comment:
Should this be required?

Reply:
There's a delivery report with status value ="Suppressed", for those events we don't have recipient mail server property, so this should be optional.